packages/cli/src/bin.ts

```diff
@@ -198,15 +198,22 @@ async function main(): Promise<number> {
         datamodel: schema,
         ignoreEnvVarErrors: true,
       })
+
       if (config.datasources.length > 0) {
         schemaProvider = config.datasources[0].provider
       }
-      const generator = config.generators.find(
-        (gen) => gen.previewFeatures.length > 0,
-      )
+
+      // restrict the search to prisma-client-js and prisma-client-go previewFeatures
+      const generator = config.generators.find((generator) => {
+        return (
+          ['prisma-client-js', 'prisma-client-go'].includes(generator.name) &&
```

Comment:
```suggestion
          ['prisma-client-js'].includes(generator.name) &&
```

Reply:
The unofficial Python client is named `prisma-client-py` and would be filtered as well as go with this PR/suggestion 

Maybe we just don't restrict the search here?

Reply:
I think we only want the preview feature of the official JS Client here. Everything else is out of our control and does not give us a clear picture.

Reply:
Then we want to keep `prisma-client-go` I would say

Reply:
See link above: https://github.com/prisma/prisma/pull/8617#issuecomment-907216484 Go Client does not support any preview features, so if someone is providing them they have no effect.

Reply:
Good point 👍🏼